action/receipt.go

```diff
@@ -13,22 +13,24 @@ import (
 	"github.com/golang/protobuf/proto"
 
 	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
 	"github.com/iotexproject/iotex-proto/golang/iotextypes"
 
 	"github.com/iotexproject/iotex-core/pkg/log"
 )
 
-// InContractTransfer is topic for system log of evm transfer
-var InContractTransfer = common.Hash{} // 32 bytes with all zeros
+// constants
+const (
+	StakingProtocolID = "staking"
```

Comment:
we may use staking.Name() if there is not conflict

Reply:
tried but unfortunately this will create import cycle